tasks/gem5art/tasks/tasks.py

```diff
@@ -37,3 +39,23 @@ def run_gem5_instance(self, gem5_run, cwd = '.'):
     """
 
     gem5_run.run(self, cwd = cwd)
+
+def run_single_job(run):
+    start_time = time.time()
+    print(f"Running {' '.join(run.command)} at {time.time()}")
+    run.run()
+    finish_time = time.time()
+    print(f"Finished {' '.join(run.command)} at {time.time()}. Total time = {finish_time - start_time}")
+
+def run_job_pool(job_list, num_parallel_jobs = mp.cpu_count() // 2):
```

Comment:
Parentheses would be helpful, if they're allowed for the default of `num_parallel_jobs`